src/reader.rs

```diff
@@ -361,12 +361,11 @@ impl Debug for GCNO {
     }
 }
 
-impl GCNO {
+impl Gcno {
     pub fn new() -> Self {
-        GCNO {
+        Gcno {
             version: 0,
             checksum: 0,
-            cwd: None,
```

Comment:
I'm not sure that's a good idea to remove this property even if it's unused.

Reply:
I stumbled over this at first and quickly got some test failures.

Everything is good as long as the value for still read [here](https://github.com/mozilla/grcov/pull/625/files#diff-267c9f4da66412a9f439ac08d224356fe24265b5e1cebb6c44c2d55b96414513R454). I simply removed it from the struct but the reading/parsing logic _should_ not be affected.

Of course I can put it back in if you prefer.

Reply:
Yep I saw that the value is still read.
It'd prefer keeping it in case we need it or even for debug purposes.

Reply:
I put it back in now. Interestingly clippy didn't warn about it anymore now 🤔